javaharness/README.md

```diff
@@ -87,6 +87,23 @@ Particles and recipes are by default loaded from the APK, but you can configure
   adb shell setprop debug.arcs.runtime.load_workstation_assets true
 ```
 
+## The Arcs Cache Manager
+The Arcs Service Manager compiles javascript and webassembly sources eagerly and caches the compiled binaries at local storage to serve subsequent requests. Page loading time and compilation overhead are optimized when enabled.
```

Comment:
Do you mean "Arcs Service Manager", or "Arcs Cache Manager"?

Reply:
Ah, typo, should be Arcs Cache Manager :)

Reply:
Fixed.